home/export_content_pages.py

```diff
@@ -147,10 +147,15 @@ def add_message_fields(self, msg_blocks: MsgBlocks) -> None:
     @staticmethod
     def serialise_buttons(buttons: blocks.StreamValue.StreamChild) -> str:
         button_dicts = []
+
         for button in buttons:
             button_dict = {"type": button.block_type, "title": button.value["title"]}
             if button.block_type == "go_to_page":
+                # Exclude buttons that has deleted pages that they are linked to it
+                if button.value.get("page") is None:
+                    continue
```

Comment:
Did we confirm that this is the desired behaviour for now, rather than failing the export with an error pointing to the problem?

Reply:
Yes for now, this is something we need to handle in the exporter. Handle it the same way we handle Related pages in the exporter. @jerith

Reply:
@jerith Yes I think this is the behaviour we want. Failing the import with an error message works because the user could give us any malformed file, and we want to tell them how to fix that file. But the export is from the database, which should only have valid data in it, so any issues with that I would consider a bug, and not something for the user to fix.

In this case we probably want to fix having null pages, but I don't know how easy a cascading delete will be here since it's not a normal table relation, but for now the short-term fix is to skip any buttons without pages (similar to if we had to cascade on delete)